importer/helpers/dagbuilder.go

```diff
@@ -403,6 +427,18 @@ func (n *FSNodeOverDag) AddChild(child ipld.Node, fileSize uint64, db *DagBuilde
 	return db.Add(child)
 }
 
+// AddChildDag adds the DAG topped by the given "child"
+// whose DAG descendents are already added into blockservice.
+func (n *FSNodeOverDag) AddChildDag(child ipld.Node, fileSize uint64, db *DagBuilderHelper) error {
+	err := n.dag.AddNodeLink("", child)
```

Comment:
Can't you just use `AddChild` everywhere? Why is there a need for original `AddChildDag`. `DagBuilderHelperInterface` is the generic form.

Reply:
The code of AddChildDag() is different than AddChild().

Reply:
ok.